magpie/__init__.py

```diff
@@ -1,273 +1 @@
-from __future__ import unicode_literals, print_function, division
```

Comment:
Why moving content of `__init__.py` into `main.py`? I am not sure there is a real benefit :)